cmake/CMakeLists.txt

```diff
@@ -1002,20 +1002,10 @@ if (CPUINFO_SUPPORTED)
   endif()
 endif()
 
-# bounds checking behavior.
```

Comment:
MS GSL doesn't provide configuration of contract violation behavior through these macros.
actually, it appears to just call std::terminate:
https://github.com/microsoft/GSL/blob/10df83d292bf5bbdc487e57dc8c2dc8c7a01f4d1/include/gsl/assert#L126-L130